aten/src/ATen/native/Linear.cpp

```diff
@@ -26,9 +26,6 @@ Tensor linear(const Tensor& input, const Tensor& weight, const c10::optional<Ten
   if (input.is_mkldnn()) {
```

Comment:
I am curious @drisspg , why the mkldnn and other backends not cleaned up as part of this PR.

Reply:
That's outside of the scope of this PR, which is focused on nested tensor support. I agree that it'd be nice to do at some point, however.